packages/nextjs/app/builders/0x119d9A1ef0D16361284a9661727b363B04B5B0c8/page.tsx

```diff
@@ -0,0 +1,178 @@
+"use client";
```

Comment:
We don't need this restriction since your page doesn't use any client-only mechanism.
Please remove this line so we can improve the app's performance by pre-rendering the page during the compile process.